docs/shipping/AWS/aws-lambda-extensions.md

```diff
@@ -245,32 +245,61 @@ If you need to use a custom pattern, you can use the environment variables `GROK
 For logs that are formatted like this:
 
 ```python
-%(app_name)s : %(message)s
+<<timestamp>> <<app_name>>: <<message>>
+# Examples
+May 04 2024 10:48:34.244 my_app: an awesome message
+May 04 2024 10:50:46.532 logzio_sender: Successfully sent bulk to logz.io, size: 472
 ```
 
-we will use `cool app` as the `app_name` and the `message` will have strings containing whitespaces, letters and numbers.
-
-In Logz.io we wish to have `app_name`, `message` in their own fields, named `my_app` and `my_message`, respectively.
+In Logz.io we wish to have `timestamp`, `app_name` and `message` in their own fields.  
 To do so, we'll set the environment variables as follows:
 
-### GROK_PATTERNS
+##### GROK_PATTERNS
+
+The `GROK_PATTERNS` variable contains definitions of custom grok patterns and should be in a JSON format.   
+- key - is the custom pattern name. 
+- value - the regex that captures the pattern.
 
-The `GROK_PATTERNS` variable should be in a JSON format.
-The key is used as the pattern name, and the value should be the regex that captures the pattern.
-In our case, while `app_name` always stays `cool app`, we don't know what `message` will be, so we need to set `GROK_PATTERNS` as: `{"app_name":"cool app","message":".*"}`
+In our example:
+- `timestamp` - matching the regex `\w+ \d{2} \d{4} \d{2}:\d{2}:\d{2}\.\d{3}`.
+- `app_name` - always a not space, so matching `\S+`.
```

Comment:
@8naama What's a not space? Should it be "Always a non-space character sequence"? Or something else?